File: AtCoderProblems/Easy/abc139/b/main.go
package main

import (
	"fmt"
)

func int2() (int, int) {
	var x, y int
	fmt.Scan(&x, &y)
	return x, y
}

func main() {
	a, b := int2()
	if b-a <= 0 {
		fmt.Println(1)
	} else {
		count := 1
		total := a
		for {
			total += (a - 1)
			count += 1
			if total >= b {
				fmt.Println(count)
				return
			}
		}
	}
}

File: AtCoderProblems/Easy/abc139/b/Main.rb
# 空白区切の入力を数値(整数)の配列で返す
def gets_i_list()
  gets.chomp.split(" ").map(&:to_i)
end

A, B = gets_i_list

if B == 1
  puts 0
else
  count = 1
  total = A
  while true
    break if total >= B
    count += 1
    total = (A * count) - (count - 1)
  end
  puts count
end
